### ISSUE TITLE
BO43 - Create utility to store a file into Firebase Storage

### ISSUE THREAD

**Issue by gregoriopalama** [gdgpescara/hedwig#242]
Create a utility method to store a file into Firebase Storage.

The utility should support the following parameters:

- file: the actual byte array of the file
- name: the name that should be used to save the file
- oldFile: the reference of the oldFile that should be deleted. This is optional

The method should return 2 informations:
- the reference to the stored file
- the public link to the stored file

**Comment by camillobucciarelli** [comment]
I want to add some development info:
- by designing the storage API if I pass the same path it overwrites the precedent file (oldFile in input isn't necessary)
- about the name we can pass the destination path (including the name of the file) and we can use it as a reference.
- the reference is a Storage type and sharing it with the app creates a binding between Astro and Firebase layer. The URL is enough, I think. If we want to do something with the file we can refer to it by the path used to store it in the storage.

In summary:
• uploadFile(file: File, destinationPath:string) : publicUrl
• deleteFile(path: string)

what do you think? @gregoriopalama
